modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/di/RepositoryProviderModule.kt

```diff
@@ -15,4 +18,19 @@ class RepositoryProviderModule {
     @Provides
     @Singleton
     fun provideTokenHandler(syncAccountManager: SyncAccountManager): TokenHandler = syncAccountManager
+
+    @Provides
+    @Singleton
+    @DownloadOkHttpClient
+    fun downloadOkHttpClient(): OkHttpClient {
```

Comment:
I am providing this in the repository module because the wear app needs the `OkHttpClient` to create a `NetworkSelectingCallFactory` which serves as the `Call.Factory` for downloads (`@DownloadCallFactory`) in the watch app. The phone and automotive apps just return this directly with their `@DownloadCallFactory` providers.